scripts/README.md

```diff
@@ -4,45 +4,34 @@
 
 A typical release goes like this:
 
-1. **Changelog**. Generate the changelog:
+### Prepare
 
-- Compare the last version with `next`. For instance: https://github.com/mui-org/material-ui/compare/v5.0.0-alpha.17...next
-- Generate the changelog with this script: https://trello.com/c/uspnIWkh/1566-release-note
-- Clean the generated changelog, to match the format of https://github.com/mui-org/material-ui/releases.
-- Open a pull request with the generated change on `next`.
+The following steps must be proposed as a pull request.
 
-1. **Build**. Prepare the release:
+1. Generate the changelog with `yarn relase:changelog`
+   The output must be prepended to the top level `CHANGELOG.md`
+   `node scripts/generateChangelog --help` for more information.
+1. Clean the generated changelog, to match the format of https://github.com/mui-org/material-ui/releases.
 
-```
-git checkout next
-git pull
-git checkout -b v5.0.0-alpha.18
-yarn
-yarn release:version
-yarn release:build
-```
-
-1. **Version**. Update the root `/package.json`'s version and commit all the changes
+1. Update the root `/package.json`'s version
 
-```
-git commit -am "v5.0.0-alpha.18"
-git tag v5.0.0-alpha.18
-```
+1. `yarn relase:version`
```

Comment:
The current behavior seems broken to me. It prompts to update every single package. Might be due to lightweight tag usage. We'll see next time if that is the case. 

If `lerna` expects to not bump any packages I would expect an explicit option for that. Right now I can only trick it by entering the same version but that's tedious.

Reply:
Lerna is configured to treat the version of the packages individually. Because of `@material-ui/types`. In this mode and from what I could gather, Lerna needs one tag per published packages in order to only bump the correct ones in the next release. This was flooding the "Tags" tab view on GitHub. The current situation is the result of a tradeoff. Bump everything in exchange for packages with multiple versions and a single tag.

Reply:
I'm not talking about tagging but updating `version` in the `package.json`. I don't understand why `lerna` thinks e.g. `@material-ui/envinfo` was changed. 

I suspect it's either caused by lightweight tags or the last tag pointing to a commit that doesn't exist on the `mui-org/material-ui` remote. Hopefully it's not just bad DX.